rudder-lang/src/ast/enums.rs

```diff
@@ -556,7 +556,8 @@ impl<'src> EnumList<'src> {
         errors
     }
 
-    /// There can be only one mapping per item that defines an identical descendant
+    // There can be only one mapping per item that defines an identical descendant
```

Comment:
It there a specific reason for this not being a doc comment? (with `///` instead of `//`)?

Reply:
Not anymore, i'm putting it back. But the following method was unused and before silencing the warning with the `allow` attribute I commented the whole function out... resulting in a compilation error (doc comment right before end of an impl block)

Reply:
:+1: